test/integration/post_controller_test.rb

```diff
@@ -1,54 +1,55 @@
 class PostControllerTest < BaseIntegrationTest
-  # This list view was started and not completed. We may come back to this post MVP
-  # test 'an authenticated user should be able to navigate to post/list successfully' do 
-  #   # Arramge
-  #   setup_session('access token', true)
+  test 'an authenticated user should be able to navigate to post/list successfully' do 
+    # Arramge
+    setup_session('access token', true)
+    GithubService.expects(:check_sse_github_org_membership).with('access token').returns(true)
 
-  #   post1 = create_post_model(title: 'title1', author: 'author1', hero: 'hero1', 
-  #                             overlay: 'overlay1', contents: 'contents1', tags: ['tag1', 'tag2'])
-  #   post2 = create_post_model(title: 'title2', author: 'author2', hero: 'hero2', 
-  #                             overlay: 'overlay2', contents: 'contents2', tags: ['tag1', 'tag2'])
-  #   GithubService.expects(:get_all_posts).with('access token').returns([post1, post2])
+    post1 = create_post_model(title: 'title1', author: 'author1', hero: 'hero1', 
+                              overlay: 'overlay1', contents: 'contents1', tags: ['tag1', 'tag2'])
+    post2 = create_post_model(title: 'title2', author: 'author2', hero: 'hero2', 
+                              overlay: 'overlay2', contents: 'contents2', tags: ['tag1', 'tag2'])
+    GithubService.expects(:get_all_posts).with('access token').returns([post1, post2])
 
-  #   # Act
-  #   get '/post/list'
+    # Act
+    get '/post/list'
 
-  #   # Assert
-  #   assert_response :success
-  # end
+    # Assert
+    assert_response :success
+  end
 
-  # test 'an authenticated user should be able to navigate to / successfully' do 
-  #   # Arramge
-  #   setup_session('access token', true)
+  test 'an authenticated user should be able to navigate to / successfully' do 
+    # Arramge
```

Comment:
Arrange? This is throughout the file

Reply:
Yes, as in the Arrange, Act, Assert format for writing unit tests. I put those comments to mark those sections in my automated tests.

Reply:
It's a typo. You have arramge with an M lol.  Looks like it's a copy-paste error throughout the file

Reply:
lol I didn't even notice that. You did say Arrange in your original comment so I was confused.